runtime/jaktlib/sorter.jakt

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2022, Rob Ryan <rob@robryan.me>
+//
+// SPDX-License-Identifier: BSD-2-Clause
+
+enum SortType {
+    Asc
+    Desc
```

Comment:
I don't think there's a reason to abbreviate those

```suggestion
    Ascending
    Descending
```

Reply:
Yeah that makes sense, this isn't sql. Updated it.